### DISCUSSION TITLE
Missing successful Preview deployment ?

### DISCUSSION THREAD

**Discussion by baixiaoyu2997** [vercel/community#1886] [Help]
### Problem Description

when i have a pr, and i already approve changes, and vercel is  deployed.  still can't be able to merge
will show:
```
Merging is blocked
Missing successful Preview deployment.
```
I googled , but I didn't find anyone with this problem

**Comment by Holatresveces** [comment]
Did you manage to solve it? I'm currently facing the same problem and cannot find anything related to this as well.

**Comment by chav-aniket** [comment]
I'm facing this issue as well :L

**Comment by Holatresveces** [comment]
Issue is still present, but repository owner was able to bypass the warning and merge the branch.

**Comment by prplecake** [comment]
I'm running into this, too, but only in one of my projects using Vercel CI.

The screenshot clearly shows GitHub knows the preview deployment is successful.
<img width="728" alt="284920000-1f35105b-64de-472f-9b2b-1ebbbca50563" src="https://github.com/vercel/community/assets/83595468/2c74231b-da0d-438d-8e40-0b81aba97f6f">

I had opened a discussion in community/community but so far haven't gotten any responses. https://github.com/orgs/community/discussions/76616

**Reply by shadoath** [reply]
Check [this](https://github.com/orgs/vercel/discussions/3215#discussioncomment-6969235) post. After I did this and pushed it to my "Preview" env, it worked.

**Reply by prplecake** [reply]
That does not help me. I am using Vercel's GitHub integration unlike the author of that post.

Vercel automatically creates the environments in GitHub for me and they're capitalized in GitHub. I appear to be having a different issue.

**Reply by jackfrosty908** [reply]
@prplecake did you manage to find out what the issue was?

**Comment by jeasmith** [comment]
I was running into this one trying to get my dependabot version bumps to auto merge. I was able to get dependabot to rebase it's open PRs by ensuring the 'Require branches to be up to date before merging' option in branch protections was enabled.

I assume that once a branch is out of date, GitHub is no longer viewing the preview deployment as valid. I can understand [why](https://john-tucker.medium.com/why-branch-is-out-of-date-is-important-6142cccdf8e6) GitHub might work like this, but the lack of documentation around this is frustrating. I have a feeling that [this piece of product feedback](https://github.com/orgs/community/discussions/48436) is also related. If you look at the screenshot in that post, you can see there's an `(outdated)` label by the deployment. I've not seen this messaging next to a deployment before and wonder if it no longer a feature to show when deployments are outdated. In any case, it makes me feel like this is the underlying reason that, despite a successful deployment, merging is not possible in these situations.
